cmd/collector/app/span_processor.go

```diff
@@ -140,7 +141,7 @@ func (sp *spanProcessor) saveSpan(span *model.Span) {
 	}
 
 	startTime := time.Now()
-	if err := sp.spanWriter.WriteSpan(span); err != nil {
+	if err := sp.spanWriter.WriteSpan(context.TODO(), span); err != nil {
```

Comment:
Using TODO here because there is a design issue with spanProcessor, which should be managing the context earlier

Reply:
Yep, for complete context propagation context needs to be added to the `processor.SpanProcessor` interface, we can do that in a follow up PR.